app/Models/Voucher.php

```
@@ -146,44 +144,49 @@ public function completeRequestStatus()
         } else {
             TransactionFlowService::updateTransactionFlow(
-                $this->id,
+                $paymentRequestId,
                 TransactionFlowName::CASH_VOUCHER_APPROVALS->value,
                 TransactionFlowStatus::DONE->value
             );
-            $this->journalEntry()->update([
+            $journalEntry->update([
                 'status' => JournalStatus::UNPOSTED->value,
             ]);
         }
```

PHPMD (2.15.0)

The method completeRequestStatus uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)